charts/stable/domoticz/questions.yaml

```diff
@@ -9,21 +9,21 @@ questions:
 # Include{replicas1}
 # Include{podSpec}
 # Include{containerMain}
-                                - variable: env
-                                  group: "App Configuration"
-                                  label: "Image Environment"
-                                  schema:
-                                    additional_attrs: true
-                                    type: dict
-                                    attrs:
-                                      - variable: WEBROOT
-                                        label: "WEBROOT"
-                                        description: "WEBROOT"
-                                        schema:
-                                          type: string
-                                          default: "domoticz"
 # Include{containerBasic}
 # Include{containerAdvanced}
+  - variable: env
+    group: "App Configuration"
+    label: "Additional Arguments"
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: EXTRA_CMD_ARG
+          label: "EXTRA_CMD_ARG"
+          description: "Append additional values for EXTRA_CMD_ARG here."
+          schema:
+            type: string
+            default: "-userdata /config"
```

Comment:
You can't expose that.
If someone changes this it will break persistence.

Reply:
Reverted the commit.